bazaarvoice/astyanax
# astyanax_jackson_library_upgrade_2.14.2

**Jira Issue**
https://bazaarvoice.atlassian.net/browse/PD-211832

**What Are We Doing Here?**
SNYK Vulnerability Remediation for Astyanax Jackson Library.
 --> Update Jackson fasterxml core (core, annotations, databind) to 2.14.2.

**How to Test and Verify**

- Check out local build : 
  Unit testing : mvn clean install

<img width="1728" alt="Screenshot 2023-06-23 at 12 40 22 PM" src="https://github.com/bazaarvoice/astyanax/assets/134580247/9cb40230-5525-4f01-9107-02a074d03c74">

**Risk**
Level : Low

**Required Testing**
Manual.

**Risk Summary**
Since risk level will be low,we will be  monitoring for vulnerability issues of jackson library in snyk dashboard.
